@utahdts/utah-design-system/@types/jsDocTypes.d.js

```diff
@@ -55,6 +55,11 @@
  * @typedef {'inline' | 'bottom' | 'bottom-left' | 'bottom-right' | 'top' | 'top-left' | 'top-right'} BannerPlacement
```

Comment:
General note: this appears to have been branched from the color picker stuff? So it has all the icons and color picker work in it that is not yet in the 2024Q3S5 branch. Maybe review those first before merging this?